py3status/modules/google_calendar.py

```diff
@@ -33,11 +32,11 @@
     format_date: The format for date related format placeholders. May be any Python
         strftime directives for dates.
         (default '%a %d-%m')
-    format_event: The format for the first toggleable event output.
-        (default '[\?color=event {summary}][ ({location})][ {format_timer}]')
-    format_event_full: The format for the second toggleable event output.
-        (default '[\?color=event {summary}]
-        [\?color=time  ({start_time} - {end_time}, {start_date})]')
+    format_event: The format for each event. The information to display for each event
+        can be toggled with 'button_toggle' based on the value of 'event_full'.
+        (default '[\?color=event {summary}][\?if=!event_full [\?if=location  ({location})]]' +\
+        '[\?if=!event_full  {format_timer}]' +\
+        '[\?if=event_full  ({start_time} - {end_time}, {start_date})]')
```

Comment:
(Note) For docs, we remove `'` on left side and `' +\`  on right side. It won't be same, but travis bot is not checking that anymore so we're just making it look nice for the docs.